packages/mysql_protocol/mysql_protocol.1.1/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+name: "mysql_protocol"
+version: "1.1"
+maintainer: "Stéphane Legrand <slegrand45@gmail.com>"
+authors: "Stéphane Legrand <slegrand45@gmail.com>"
+homepage: "https://github.com/slegrand45/mysql_protocol"
+bug-reports: "https://github.com/slegrand45/mysql_protocol/issues"
+license: "LGPL"
+dev-repo: "https://github.com/slegrand45/mysql_protocol"
```

Comment:
Could you please change the `dev-repo` field to

```
dev-repo: "https://github.com/slegrand45/mysql_protocol.git"
```

to help the [OPAM 1.2.0 parser](https://travis-ci.org/ocaml/opam-repository/jobs/87790071#L380) along?